viz/cmd/stat.go

```diff
@@ -839,6 +844,11 @@ func buildStatSummaryRequests(resources []string, options *statOptions) ([]*pb.S
 		}
 		requests = append(requests, req)
 	}
+
+	if usingTs {
+		fmt.Printf("Native support for SMI resources is deprecated and will be removed in 2.12. Please follow the SMI extension getting started guide from https://linkerd.io/2.10/tasks/linkerd-smi/\n")
```

Comment:
I think the wording here is fine. I was going to suggest maybe prefixing with some like "Warning:", but we don't do that for recently moved `linkerd viz` commands: `Command "stat" is deprecated ...`

If anything I would at least include a new line so that the "warning" is separate from the command output.

Reply:
Makes sense. Added a new `\n` at the end to make it look like the cobra's deprecated cmd warning